deltachat-ios/ViewModel/ChatListViewModel.swift

```diff
@@ -42,10 +16,10 @@ class ChatListViewModel: NSObject, ChatListViewModelProtocol {
         case messages
     }
 
-    var isArchive: Bool
+    private(set) public var isArchive: Bool
```

Comment:
interesting, I haven't seen that syntax yet, a little bit confusing to read though.

is there a real reason to change the visibility here?

Reply:
i tried just to do was the protocol was doing - making `isArchive` public read and private read/write.

the syntax `private(set) public` is already used in some pods, so if you grep for it, you will find some things.

but i would also be fine in just making it `public` and not introduce a new syntax variant.